src/widgets/SingleLineTextInput.js

```diff
@@ -87,6 +87,15 @@ export default class SingleLineTextInput extends createjs.Container {
     this._mode = MODES.INSERT;
   }
 
+  get text() {
+    return this._text.text;
```

Comment:
Since we are already using loadash, would it be better to use _.get() to avoid the possible "..... text of undefined" error? (will also allow for an explicit default to be set)

Reply:
That case should never happen since `this._text` is set to a createjs.Text instance in the constructor and never changes (just its fields get updated).  If null or undefined where possibilities, then I would agree with `_.get`.